src/js/api/auth/logout.test.js

```diff
@@ -0,0 +1,20 @@
+import { logout } from "./logout.js";
+import { remove } from "../../storage/index.js";
+
+// Mock
+jest.mock("../../storage/index.js", () => ({
+  remove: jest.fn(),
+}));
+
+describe("logout function", () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("should clear the token from the browser", () => {
+    logout();
```

Comment:
This test will always pass, since you never login first. You need to login to logout